scripts/mailing_lists/migrate_project_mailing_lists.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
```

Comment:
In order to run these scripts, there needs to be an `__init__.py` file in `scripts/mailing_lists/`.